tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -130,7 +130,7 @@ public function testGeneratesForeignKeySqlForNonStandardOptions(): void
             'my_table',
             ['id'],
             'my_fk',
-            ['feferred' => true],
+            ['deferred' => true],
```

Comment:
This will be taken care of in 4.0.x (https://github.com/doctrine/dbal/pull/3520) but for now it needs to be kept as is to guarantee backward compatibility.

Reply:
Committed new PR reverting to the old